### DISCUSSION TITLE
Support for `mailto:` in `./github/ISSUE_TEMPLATE/config.yml`?

### DISCUSSION THREAD

**Discussion by Meiryo7743** [community/community#23971] [Projects and Issues]
<p>I put <code>config.yml</code> in <code>./github/ISSUE_TEMPLATE</code> for the purpose of adding an e-mail address so that users can ask anything but technical topics.<br>
However, the additional menu doesn’t show on the creating a new issue page.</p>
<p>I have made sure it can be displayed correctly if <code>url</code> is a URL starting with <code>http://</code> or <code>https://</code>. Also I read <a href="https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser" rel="noopener nofollow ugc">the docs</a>, but there is no clue …</p>
<p>So I have a question: is <code>url</code> supported a URL whose prefix is <code>mailto:</code>? If not, I think it would be nice to support such links.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>No, the format doesn’t allow for a mailto or direct email address as a url, but there are online services that will convert your desired mailto format to a shortened url, and those will work provided the person has a compatible email client setup.</p>
<p>I can’t vouch for any of those services in particular but you’ll find a few to evaluate if you search for a <code>mailto link shortener</code>.</p>

**Comment by Meiryo7743** [comment]
<p><a class="mention" href="/canuckjacq">@canuckjacq</a></p>
<p>That’s a good idea!</p>
<p>I didn’t know <code>mailto:</code> links can also be shorten like URLs, and will try the solution.</p>
<p>Thank you for your helpful advice.</p>
